content/blog/automation-api-as-platform/index.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "Build Self-Service Cloud Infrastructure with Automation API "
+date: 2020-11-09
+meta_desc: "With Pulumi's Automation API, you can build a REST API for creating, reading, updating, and deleting (CRUD) cloud resources."
+meta_image: automation_api.png
+authors:
+    - sophia-parafina
+tags:
+    - Automation API
+    - CRUD
+---
+
+If you could create infrastructure without using a cloud provider's console, a CLI, or a templating engine, what would you build? Pulumi's Automation API lets you create declarative infrastructure defined by your best practices and expose it behind a REST, gRPC, or custom API.
+
+So just what is Automation API? Think of it as Pulumi's infrastructure as code engine as an SDK. Instead of writing code and using the CLI to declare infrastructure, you can directly tell the engine to build your infrastructure. Imagine building an application that creates infrastructure via a REST interface. Get ready, because that's what we're going to do.
+
+<!--more-->
+
+## Self-service Cloud Infrastructure
+
+Automation API is just another package that runs inside your favorite frameworks and works with other packages. You can create methods or functions to create infrastructure that you can call from other frameworks. One Automation API use case is self-service infrastructure. You can create declarative infrastructure and expose it behind a REST interface.
```

Comment:
> Automation API is just another package that runs inside your favorite frameworks and works with other packages. You can create methods or functions to create infrastructure that you can call from other frameworks

It might be worth calling out that the end result is still a Pulumi `stack` backed by JSON state file managed by the Pulumi service. The benefit here is being able to create, and update them dynamically and on the fly.

Reply:
added the point